simplex-solver/build.boot

```diff
@@ -0,0 +1,61 @@
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all]
+         '[boot.core :as boot]
+         '[boot.tmpdir :as tmpd]
+         '[clojure.java.io :as io]
+         '[boot.util :refer [sh]]
+         '[clojure.string :as string])
+
+(def +lib-version+ "0.0.3")
+(def +version+ (str +lib-version+ "-0"))
+(def +commit-id+ "8e55b56d499dc7d64ac22a5051f2eeb4a22a2534")
+(def +src-url+ "https://github.com/SamDuvall/simplex-solver")
+(def +lib-folder+ (format "simplex-solver-%s" +commit-id+))
+
+(task-options!
+ pom  {:project     'cljsjs/simplex-solver
+       :version     +version+
+       :description "A Simple Simplex Solver"
+       :url         +src-url+
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+(deftask download-solver []
+  (let [url (str +src-url+ "/archive/" +commit-id+ ".zip")
```

Comment:
There is no `+lib-version+` zip available in the repo. That's why I used the commit id instead.